src/qibo/gates/gates.py

```diff
@@ -337,7 +369,7 @@ class _Rn_(ParametrizedGate):
     Args:
         q (int): the qubit id number.
         theta (float): the rotation angle.
-        trainable (bool): whether gate parameters can be updated using
+        trainable (bool, optional): whether gate parameters can be updated using
```

Comment:
Here I would prefer to leave it as it is (`bool` only), because `optional` usually means that it can be `None` (but I may be wrong), which is not the case here. `trainable` can only be `True` or `False`.